make-go.bash

```diff
@@ -50,10 +66,20 @@ if [ -d "$GOROOT_FINAL" ]; then
 fi
 mkdir -p "$GOROOT_FINAL"
 
+# Fetch latest Go
+if [ ! -f go-latest.tar.gz ]; then
+  wget -O go-latest.tar.gz $GO_SOURCE_URL$GO_SOURCE
```

Comment:
I actually don't fully agree with this change. Readability should highly be optimized in a bash script and all these variables make it harder to read imo. But updated as requested..